app/screens/AppNavigator/screens/Dex/DexScreen.tsx

```diff
@@ -195,3 +200,19 @@ function PoolPairInfoLine (props: { symbol: string, reserve: string, row: string
     </View>
   )
 }
+
+function PoolPairAPR (props: { symbol: string, apr: number, row: string }): JSX.Element {
+  return (
+    <View style={tailwind('flex-row justify-between items-end')}>
+      <Text style={tailwind('text-sm font-semibold mb-1')}>{translate('screens/DexScreen', 'APR')}</Text>
+      <NumberFormat
+        suffix='%'
+        value={new BigNumber(isNaN(props.apr) ? 0 : props.apr).times(100).toFixed(2)} decimalScale={2} thousandSeparator
```

Comment:
for `number` that comes differently from whale API that isn't encoded in string, it means you can use the number as it is because it's not precision sensitive